GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml

```diff
@@ -0,0 +1,257 @@
+        app:layout_constraintBottom_toTopOf="@+id/item13"
+        app:layout_constraintEnd_toStartOf="@+id/item10"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/item05" />
+
+    <TextView
+        android:id="@+id/item10"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorJ"
+        android:text="J"
+        android:textColor="@android:color/white"
+        android:textSize="50sp"
+        android:gravity="center_horizontal|center_vertical"
+        app:layout_constraintBottom_toTopOf="@+id/item14"
+        app:layout_constraintEnd_toStartOf="@+id/item11"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item09"
+        app:layout_constraintTop_toBottomOf="@+id/item06" />
+
+    <TextView
+        android:id="@+id/item11"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorK"
+        android:text="K"
+        android:textColor="@android:color/white"
+        android:textSize="50sp"
+        android:gravity="center_horizontal|center_vertical"
+        app:layout_constraintBottom_toTopOf="@+id/item15"
+        app:layout_constraintEnd_toStartOf="@+id/item12"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item10"
+        app:layout_constraintTop_toBottomOf="@+id/item07" />
+
+    <TextView
+        android:id="@+id/item12"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorL"
+        android:text="L"
+        android:textColor="@android:color/white"
+        android:textSize="50sp"
+        android:gravity="center_horizontal|center_vertical"
+        app:layout_constraintBottom_toTopOf="@+id/item16"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item11"
+        app:layout_constraintTop_toBottomOf="@+id/item08" />
+
+    <TextView
+        android:id="@+id/item13"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorM"
+        android:text="M"
+        android:textColor="@android:color/white"
+        android:textSize="50sp"
+        android:gravity="center_horizontal|center_vertical"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/item14"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/item09" />
+
+    <TextView
+        android:id="@+id/item14"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorN"
+        android:text="N"
```

Comment:
extracting text to string file may be a good idea to reuse it

Reply:
Done.